packages/portal/src/server-middleware/api/jira-service-desk/feedback.js

```diff
@@ -39,13 +39,19 @@ const jiraData = (options, req) => {
 
 const validateFeedbackLength = feedback => wordLength(feedback) >= 5;
 
-const validateFeedback = feedback => new Promise((resolve, reject) => {
-  if (validateFeedbackLength(feedback)) {
-    resolve();
-  } else {
-    reject(createHttpError(400, 'Invalid feedback.'));
-  }
-});
+const validateHost = (host, allowedHosts) => {
+  return allowedHosts.some(allowedHost => host === allowedHost);
+};
+
+const validateFeedback = (requestBody, allowedHosts) => {
+  return new Promise((resolve, reject) => {
+    if (validateFeedbackLength(requestBody.feedback) && validateHost(new URL(requestBody.pageUrl).host, allowedHosts)) {
+      resolve();
+    } else {
+      reject(createHttpError(400, 'Invalid feedback.'));
```

Comment:
Do we maybe want a bit clearer error handling here? For example defining which validation failed.